llvm/test/Verifier/target-ext-vector.ll

```diff
@@ -0,0 +1,11 @@
+; RUN: llvm-as -o - %s | llvm-dis | FileCheck %s
+
+; CHECK-LABEL: @vec_ops(
+define <2 x target("spirv.Image")> @vec_ops(<2 x target("spirv.Image")> %x) {
```

Comment:
This probably shouldn't be *actually* valid, right? Maybe we should have a separate extension type property that decides whether it can be used in a vector or not?